addon/lib/StudyTelemetryCollector.jsm

```diff
@@ -33,47 +32,50 @@ XPCOMUtils.defineLazyModuleGetter(this, "PlacesUtils",
 XPCOMUtils.defineLazyServiceGetter(this, "Telemetry",
   "@mozilla.org/base/telemetry;1", "nsITelemetry");
 
+// unit-tested study helpers
+XPCOMUtils.defineLazyModuleGetter(
+  this, "Helpers", "resource://esper-pioneer-shield-study/lib/Helpers.jsm"
+);
+
 /**
  * Note: Setting studyUtils in this constructor due to above comments
  */
 class StudyTelemetryCollector {
 
-  constructor(studyUtils, variation) {
-    this.variation = variation;
-    this.studyUtils = studyUtils;
+  constructor(pioneerUtils) {
```

Comment:
you shouldn't need to pass `pioneerUtils` in here since you can just import `Pioneer.jsm` above and use `Pioneer.utils`.

Reply:
Probably yes, but had issues import ShieldUtils previously which is why I chose to send it as part of the constructor. Trying to minimize changes in the add-on to avoid unintended bugs from refactoring. Will use the feedback for the next pioneer add-on.

Reply:
Fixed :)